src/app/edit-profile/edit-profile.page.html

```diff
@@ -41,12 +44,22 @@
             </ion-item>
 
             <ion-item [hidden]="needVerification">
+              <ion-label position="fixed">
+                Sex
+              </ion-label>
+              <ion-select placeholder="Select One" name="sex" [(ngModel)]="sex" required>
+                <ion-select-option value="Female">Female</ion-select-option>
+                <ion-select-option value="Male">Male</ion-select-option>
+              </ion-select>
+            </ion-item>
+
+            <!-- <ion-item [hidden]="needVerification">
```

Comment:
Can delete?

Reply:
Yep, I noticed that too and I've deleted them. Thanks!